apps/docusaurus/docs/designsystems/Komponenter/checkbox.md

```diff
@@ -36,18 +39,122 @@ Avmerkingsboksen får fokus når den velges med tastaturet.
 
 ## Feilmelding avmerkingsboks
 
-Følgende avmerkingsboks får feilmeldingsfarge hvis ett av alternativene må være valgt før bruker kan gå videre.
+Følgende avmerkingsboks får feilmeldingsfarge og en feilmelding hvis ett av alternativene må være valgt før bruker kan
+gå videre.
 
-<div class="component__display">
+<div class="component__display__column">
     <div class="kv-checkbox">
         <input type="checkbox" id="checkbox_2" required/>
-        <label class="detail detail--sml" for="checkbox_2">Input tekst</label>
+        <label class="detail detail--sml" for="checkbox_2">Jeg aksepterer vilkårene for...</label>
+        <div class="error__box">
+            <i class="material-symbols-outlined">exclamation</i>
```

Comment:
Her, og i alle de andre error-boxene skal det i følge figma-filen være brukt symbolet "error", i stedet for "exclamation", da får utropstegnet en ring rundt seg, slik som i designet.

Reply:
Dette lærte jeg på fredag av Frida da jeg arbeidet med radioknapp, har bare glemt å oppdatere de gamle oppgavene 😅.